CODE_OF_CONDUCT.md

```diff
@@ -1,105 +0,0 @@
-<!--
```

Comment:
Maybe keep the file as it's a top-level file and just have a link to the ASF CoC?